src/measures.jl

```diff
@@ -83,6 +83,23 @@ function measure(t::Tetrahedron)
   abs((A - D) ⋅ ((B - D) × (C - D))) / 6
 end
 
+"Compute the measure (i.e. hyper-volume) for a simplex in any dimension using the Cayley-Menger Determinant."
+function measure(splx::Simplex{K,Dim,T,N}) where {K, Dim, T<:Real, N}
+  distancebuf = Matrix{T}(undef, N+1, N+1)
+  measure!(distancebuf, splx)
+end
+
+"In-place verion of [`measure`](@ref)."
+function measure!(distancebuf::AbstractMatrix{T}, splx::Simplex{K,Dim,T,N}) where {K, Dim, T<:Real, N}
+  size(distancebuf) == (N+1, N+1) || throw(ArgumentError("buffer dimensions must be (N+1) times (N+1)"))
+  # https://en.wikipedia.org/wiki/Cayley%E2%80%93Menger_determinant
+  pairwise!(SqEuclidean(), @view(distancebuf[1:end-1, 1:end-1]), coordinates.(vertices(splx)))
+  distancebuf[end, 1:end-1] .= 1; distancebuf[1:end-1, end] .= 1; ; distancebuf[end, end] = 0
+  factor = (-1)^(K+1)/(factorial(K)^2*2^K)
+  return sqrt(factor * det(distancebuf))
+end
```

Comment:
Please avoid introducing a new function `measure!`. Our interface for measure takes a geometry and returns a scalar, there is no need for a variant with mutation.